modules/installing-rhv-setting-up-installation-machine.adoc

```diff
@@ -7,7 +7,7 @@
 = Setting up the installation machine
 
 
-To run the binary `openshift-install` installation program and Ansible scripts, set up the {rh-virtualization} Manager or an {op-system-base-full} computer with network access to the {rh-virtualization} environment and the REST API on the {rh-virtualization} Manager/oVirt Engine.
+To run the binary `openshift-install` installation program and Ansible scripts, set up the {rh-virtualization} Manager or an {op-system-base-full} computer with network access to the {rh-virtualization} environment and the REST API on the {rh-virtualization-engine-name}.
```

Comment:
Do you need to get this one: `{rh-virtualization} Manager`?

Reply:
Yes, thanks for the catch. I just changed a bunch of those.